ew/cmd/juviecmd/juviecmds.py

```diff
@@ -407,8 +407,10 @@ async def mine(cmd):
                 weapon = static_weapons.weapon_map.get(weapon_item.item_props.get("weapon_type"))
                 if (weapon.id_weapon == ewcfg.weapon_id_pickaxe or weapon.id_weapon == ewcfg.weapon_id_diamondpickaxe) and user_data.life_state != ewcfg.life_state_juvenile:
                     toolused = "pickaxe"
+                elif (weapon.id_weapon == ewcfg.weapon_id_shovel):
+                    toolused = ewcfg.weapon_id_shovel
                 elif weapon.id_weapon == ewcfg.weapon_id_sledgehammer:
-                    toolused = "sledgehammer"
+                    toolused = ewcfg.weapon_id_sledgehammer
```

Comment:
Please use your brain and actually don't do this. Doing this will break the rest of the code pertaining to the sledgehammer.